Makefile

```diff
@@ -151,8 +156,11 @@ tidy-deps:
 
 .PHONY: $(GO_APP_BUILD_TARGETS)
 $(GO_APP_BUILD_TARGETS): build-%:
+	$(eval VERSION := $(shell git describe --tags --always --abbrev=7))
```

Comment:
Maybe we could create a rule in the Makefile to generate the version.
I'm saying this because we have many places using the same code to generate it.
It doesn't need to be done in this PR, but let's keep this in mind.